.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+version: 2.1
+orbs:
+  slack: circleci/slack@3.4.2
+jobs:
+  generate docs:
+    docker:
+      - image: cimg/node:lts
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
+    resource_class: xlarge     
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: npm install
+          command: sudo npm install -g lerna && npm install
+      - run:
+          name: pull Tymly monorepo
+          command: git clone https://github.com/wmfs/tymly.git
+      - run:
+          name: sync Tymly monorepo
+          command: |
+            cd tymly
+            npm install
+            TYMLY_GITHUB_USER=$GIT_AUTHOR_NAME TYMLY_GITHUB_TOKEN=$GH_TOKEN npm run sync public-only
+      - run:
+          name: bootstrap Tymly monorepo
+          command: |
+            cd tymly
+            lerna clean --yes
+            lerna bootstrap -- --production
+            lerna link --force-local
+      - run:
+          name: generate docs
+          command: |
+            cd tymly/packages/tymly-doc-generator
+            TYMLY_MONOREPO_PATH=/home/circleci/repo/tymly TYMLY_DOCS_OUTPUT_PATH=/home/circleci/repo npm run generate
+      - run:
+          name: deploy docs
+          command: |
+            git config --global user.name "$GIT_COMMITTER_NAME"
+            git config --global user.email "$GIT_COMMITTER_EMAIL"
+            echo "Building"
+            npm run docs:build
+            cd docs/.vuepress/dist
+            echo "Deploying"
+            git init
+            git add -A
+            git commit -m "deploy [skip ci]"
```

Comment:
@meenahoda 

Update commit message to follow conventional commit style.  "feat: deploy docs [skip ci]" or something.